noxfile.py

```diff
@@ -254,9 +254,10 @@ def run_tests(session):
     session.install("setuptools", silent=False)
     session.install("-e", ".[all,dev,jax]", silent=False)
     specific_test_files = session.posargs if session.posargs else []
```

Comment:
couldn't this be simplified with this:
```suggestion
    specific_test_files = session.posargs if session.posargs else ["unit or integration"]
```

Reply:
No, I don't think so – for that we'll need the `-m` flag, but we exclude that when invoking a test file directly